src/javascript/db.js

```diff
@@ -128,10 +148,10 @@ export function updateSong(info) {
 }
 
 function cleanForUpdate(obj) {
-  obj.dateUpdated = dbi.Timestamp.now();
+  obj.dateUpdated = firestore.Timestamp.now();
   // FIXME handle this logic better
   for (let propName in obj) {
-    if (obj[propName] === null || obj[propName] === undefined) {
+    if (obj[propName] === undefined) {
       delete obj[propName];
     }
   }
```

Comment:
You can maybe do

```js
Object.keys(obj).forEach(key => {
  if (obj[key] === undefined) {
    delete obj[key];
  }
});
```

Which basically is the same as the logic here.

Reply:
Cool! That's right! fixed.